packages/cloud-cognitive/src/components/CreateTearsheet/CreateTearsheet.js

```diff
@@ -32,7 +32,7 @@ import {
 } from '../../global/js/hooks';
 import { hasValidChildType } from '../../global/js/utils/hasValidChildType';
 
-const componentName = 'CreateTearsheet';
+export const componentName = 'CreateTearsheet';
```

Comment:
Nothing wrong with this per se, but we don't actually need to do this as the component name is attached to the component too.

```suggestion
```